src/renderer/components/basic.lua

```diff
@@ -148,27 +174,31 @@ function basicTextComponent:resolveHeight(styles, context, resolver)
     local img = surface.load(path)
 
     cY = math.ceil(img.height / 3)
-  elseif styles["font-size"] == "2em" then
+  elseif styles["font-size"] == "2rem" then
     cY = math.ceil(font.height / 3)
-  elseif styles.content then
-    cY = util.wrappedWrite(nil, resolver({}, "string", styles.content),
-      0, cY, context.width - leftPad - rightPad)
-  else
-    local children = self.node.children
-    local acc = ""
-    foreach(child, children) do
-      if child.type == "text" then
-        acc = acc .. child.content
-      elseif child.name == "br" then
-        cY = util.wrappedWrite(nil, acc,
-          position.left + leftPad, cY, position.width - leftPad - rightPad)
-        acc = ""
-        cY = cY + 1
-      elseif child.name == "span" then
-        acc = acc .. child.children[1].content
+  else styles["font-size"] == "1rem" or totalSize <= 1 then
+    if styles.content then
+      cY = util.wrappedWrite(nil, resolver({}, "string", styles.content),
+       0, cY, context.width - leftPad - rightPad)
+    else
+      local children = self.node.children
+      local acc = ""
+      foreach(child, children) do
+        if child.type == "text" then
+          acc = acc .. child.content
+        elseif child.name == "br" then
+          cY = util.wrappedWrite(nil, acc,
+            position.left + leftPad, cY, position.width - leftPad - rightPad)
+          acc = ""
+          cY = cY + 1
+        elseif child.name == "span" then
+          acc = acc .. child.children[1].content
+        end
       end
+      cY = cY + 1
     end
-    cY = cY + 1
```

Comment:
Uhh, it needs to be incremented regardless of whether the text was in content, or not.

Reply:
That was a pure mistake, forgive me.